class.jetpack.php

```diff
@@ -669,6 +670,15 @@ function point_edit_links_to_calypso( $default_url, $post_id ) {
 		return esc_url( sprintf( 'https://wordpress.com/%s/%s/%d', $path_prefix, $site_slug, $post_id ) );
 	}
 
+	function point_edit_comment_links_to_calypso( $link, $comment_id, $text ) {
+		$url = wp_parse_url( get_home_url() );
```

Comment:
I'm pretty sure this is going to break on Jetpack sites with subfolder installs. I suggest following the existing pattern in `point_edit_post_links_to_calypso` to avoid that:

`$site_slug  = Jetpack::build_raw_urls( get_home_url() );`